Fix for Solaris.
This should be investigated for sending upstream.

--- thunderbird-127.0/comm/mail/app/profile/all-thunderbird.js.old	2024-06-08 21:24:15.119045011 -0400
+++ thunderbird-127.0/comm/mail/app/profile/all-thunderbird.js	2024-06-08 21:24:32.042611175 -0400
@@ -193,7 +193,7 @@
 // Disable add-ons installed into the shared user and shared system areas by
 // default. This does not include the application directory. See the SCOPE
 // constants in AddonManager.sys.mjs for values to use here
-pref("extensions.autoDisableScopes", 15);
+pref("extensions.autoDisableScopes", 11);
 
 // Enable add-ons installed and owned by the application, like the default theme.
 pref("extensions.startupScanScopes", 4);
